.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+## Thank you for contributing to Spellbook 🪄
+
+### Contribution type
+Please check the type of contribution this pull request is for:
+
+- [ ] New spell(s)
+- [ ] Adding to existing spell lineage
+- [ ] Bug fix
+
+---
+
```

Comment:
maybe we add a little note that they should discard the sections below that don't apply

Reply:
good idea